code/processes/dqe.q

```diff
@@ -113,8 +113,22 @@ postback:{[runtype;idnum;proc;params;result]
     .dqe.updresultstab[runtype;idnum;.z.p;first result;result[1];`complete;params;proc]];
   }
 
+containerfn:{[funct;names;vars]                                                                                 /- used to check if table is being called in HDB
+  if[any b:where 11h=abs type each vars;                                                                        /- checks if any variable for check function is type symbol
+    startvars:vars b;
+    names:names b;
+    finvars:startvars b2:where startvars in .Q.pt;                                                              /- check if the variables are partitioned tables
+    names:names b2;
+    if[count finvars;
+      names set ' value each ("select from " ,/: (string finvars)),\:" where date=.z.p-1"];                     /- replace table names with select statments when run in HDB
```

Comment:
Not sure about creating strings here, can functional form be used? Seems like it could be a good fit.